app/ide-desktop/lib/dashboard/src/authentication/src/error.ts

```diff
@@ -29,6 +33,39 @@ export function tryGetMessage(error: unknown): string | null {
         : null
 }
 
+/** Like {@link tryGetMessage} but returns the string representation of the value if it is not an
+ * error.
+ *
+ * @param error - an {@link Error} or any other value.
```

Comment:
i *think* we're supposed to avoid `@param` and `@returns` because parameter names should be more or less self-explanatory, so usually a `@param` would be redundant.

in this case you can also see that the `@returns` just re-states the description on the first line

Reply:
Fixed.